packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
Yeah i also thought about that – we could merge those to `aggregateId`, because that's what it is. But not sure if this really makes things easier to understand … i guess people are just confused about CQRS concepts at first.